dataflow/src/org/checkerframework/dataflow/analysis/AnalysisResult.java

```diff
@@ -31,6 +32,9 @@
     /** Map from AST {@link Tree}s to {@link Node}s. */
     protected final IdentityHashMap<Tree, Node> treeLookup;
 
+    /** Map from AST {@link UnaryTree}s to compound {@link AssignmentNode}s. */
+    protected final IdentityHashMap<UnaryTree, AssignmentNode> compoundAssignTreeLookupMap;
```

Comment:
Why do we need a separate map for the UnaryTree? Can the same UnaryTree be both in `treeLookup` and in `compoundAssignTreeLookupMap`? If so, describe what the two different values mean. If not, why have the separate map?